garage/misc/logger/tensorboard_output.py

```diff
@@ -128,6 +131,18 @@ def record_tensor(self, key, val):
             self._scalars.value.add(
                 tag=key + '/' + str(idx).strip('()'), simple_value=float(v))
 
+    def record_matplotlib(self, key, fig):
+        height, width = fig.get_size_inches() * fig.dpi
+        image_buffer = io.BytesIO()
+        fig.savefig(image_buffer, format='png')
```

Comment:
thanks for using compressed images (it still takes up a lot of space for 400 images in the example runner)

Reply:
SVG would be tiny because it's a vector format. Bitmap by far the largest because it is lossless and unencoded. PNGs very but sometimes use internal lossless compression (LZ i *think*)